test/reactTags.test.js

```diff
@@ -137,7 +137,7 @@ describe('Test ReactTags', () => {
       mockItem({
         delimiters: [Keys.TAB, Keys.SPACE, Keys.COMMA],
         handleAddition(tag) {
-          actual.push(tag);
+          actual.push(tag.text);
```

Comment:
Instead of changing it text here can you fix the spec so that it compares array of objects